content/python/concepts/functions/functions.md

```diff
@@ -113,9 +130,45 @@ def multiply(*args):
     product *= arg
   return product
 
-multiply(21, 24, 17)
+multiply(21, 24)           # Output: 504
+multiply(10, 5, 3, 6, 17)  # Output: 15300
+```
+
+Similarly, functions can be called with an arbitrary number of keyword arguments. In this case, a special parameter `**kwargs` is passed in, where the double asterisk is a packing operator that produces a [dictionary](https://www.codecademy.com/resources/docs/python/dictionaries) rather than a tuple. The parameter name and value of each keyword argument are packed as a key-value pair stored in `kwargs`.
+
+```py
+def north_american_leaders(**kwargs):
```

Comment:
Can we change this snippet to not include political figures? Maybe change it to `north_american_capitals`? And let's also rewrite the output snippet like suggested previously:

```py
def north_american_capitals(**kwargs):
  ...

north_american_capitals(canada="Ottawa", us="Washington D.C.", mexico="Mexico City")
```

The output of the function call will be:

```
canada: Ottawa
us: Washington D.C.
mexico: Mexico City
```